### DISCUSSION TITLE
Man page

### DISCUSSION THREAD

**Discussion by elkarouh** [eza-community/eza#407] [Q&A, Howto, Help] [answered]
This should be posted in Q&A in discussions
I installed the tool with 'cargo install eza'
But how can i install the man pages?

**Comment by cafkafk** [comment]
Are you on Linux, MacOS, or windows? If Linux, what distribution are you on?

**Reply by elkarouh** [reply]
Red Hat 7

**Reply by cafkafk** [reply, marked as answer]
Here are the manpages, you might be able to extract them with `tar xf eza_manpages.tar.gz` and then move them into `/usr/share/man` for now.
[eza_manpages.tar.gz](https://github.com/eza-community/eza/files/12700029/eza_manpages.tar.gz)

We'll probably ship manpages with the release soon.

**Comment by elkarouh** [comment]
Many thanks

**Reply by cafkafk** [reply]
NW! remember to mark the answer so others can find it easily in the future.
